cmd/flipt/main.go

```diff
@@ -59,7 +63,9 @@ var (
 		EncodeDuration: zapcore.StringDurationEncoder,
 		EncodeCaller:   zapcore.ShortCallerEncoder,
 	}
-	fatal = zap.Must(defaultConfig(defaultEncoding).Build()).Fatal
+	defaultLogger   = zap.Must(defaultConfig(defaultEncoding).Build())
+	userHomeDir, _  = os.UserHomeDir()
+	fliptConfigFile = fmt.Sprintf("%s/.flipt/config.yml", userHomeDir)
```

Comment:
Sorry, there was one thing I just sort of glazed over when I originally approved:

In the original proposal I suggested the following location:
```
{{ os.UserConfigDir() }}/flipt/config.yml
```

This change suggests `$HOME/.flipt/config.yml`. I wonder if it would be more inline to put config in the systems config directory?

We're already putting our telemetry in here:
https://github.com/flipt-io/flipt/blob/main/cmd/flipt/main.go#L352-L358

So now we could have those files side by side and only create one directory for the system.

Reply:
👍🏻 agreed